### ISSUE TITLE
FID score on Imagenet-256 dataset

### ISSUE THREAD

**Issue by cientgu** [openai/guided-diffusion#14]
It's really a great work! However, on ImageNet-256 dataset, I test the FID between 50k images from the validation set and 50k images from the training set, the FID is about 7.6. (using two popular FID code, https://github.com/mseitzer/pytorch-fid and https://github.com/tsc2017/Frechet-Inception-Distance , which get very similar results). So I am really confused how to achieve the FID score 4.59 in your paper. Could you share the code to calculate the FID score? I just want to make sure my way to calculate FID is not wrong.

**Comment by je-santos** [comment]
I also got ~7.5 using 10k images. @cientgu any luck?

**Comment by Yuxin-Zhang-Jasmine** [comment]
There is an evaluator.py in the folder evaluations

**Comment by sunset-clouds** [comment]
The rFID on the Imagenet-256 I got from the test is 2.70, but many people told me that the rFID of VAR is 0.92. I don’t know which step in the test went wrong.

Inception Score: 56.86065673828125
FID: 2.70789722116308
sFID: 4.6903826389984715
Precision: 0.74194
Recall: 0.6662
